server/controllers/lectures.ts

```diff
@@ -0,0 +1,61 @@
+import Lecture from "../Lecture.ts";
+
+const lectures = new Map();
+
+const createLecture = async ({
+  request,
+  response,
+}: {
+  request: any;
+  response: any;

```

Comment:
skoro zdecydowaliśmy się na TS to jeśli "any" nie jest koniecznością to należy korzystać z odpowiednich typów bo inaczej TS nie ma sensu

Reply:
t, to pozostałośc po straym kodzie, rzeczywiscie trzeba zmienic na string i string